<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstTransactionPage</class>
 <widget class="QDialog" name="FirstTransactionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frameImport">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>1. Import Bitcoin private keys</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="editImportKey">
            <property name="placeholderText">
             <string>private key exported by 'dumpprivkey'</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonImportKey">
            <property name="text">
             <string>Import Key</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Balance: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelBalance">
            <property name="text">
             <string>125.000 BTC</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRescan">
            <property name="text">
             <string>Rescan</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameCoinControl">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutCoinControl" stretch="0,0,0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutCoinControl1">
            <property name="bottomMargin">
             <number>15</number>
            </property>
            <item>
             <widget class="QLabel" name="labelCoinControlFeatures">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">font-weight:bold;</string>
              </property>
              <property name="text">
               <string>2. Select Bitcoin Inputs to transfer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutCoinControl2" stretch="0,0,0">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButtonCoinControl">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Inputs...</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelCoinControlInsuffFunds">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:red;font-weight:bold;</string>
              </property>
              <property name="text">
               <string>Insufficient funds!</string>
              </property>
              <property name="margin">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacerCoinControl">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widgetCoinControl" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayoutCoinControl5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayoutCoinControl3" stretch="0,0,0,1">
               <property name="spacing">
                <number>20</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <layout class="QFormLayout" name="formLayoutCoinControl1">
                 <property name="horizontalSpacing">
                  <number>10</number>
                 </property>
                 <property name="verticalSpacing">
                  <number>14</number>
                 </property>
                 <property name="leftMargin">
                  <number>10</number>
                 </property>
                 <property name="topMargin">
                  <number>4</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="labelCoinControlQuantityText">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Quantity:</string>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="labelCoinControlQuantity">
                   <property name="cursor">
                    <cursorShape>IBeamCursor</cursorShape>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::ActionsContextMenu</enum>
                   </property>
                   <property name="text">
                    <string notr="true">0</string>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="labelCoinControlBytesText">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Bytes:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="labelCoinControlBytes">
                   <property name="cursor">
                    <cursorShape>IBeamCursor</cursorShape>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::ActionsContextMenu</enum>
                   </property>
                   <property name="text">
                    <string notr="true">0</string>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayoutCoinControl2">
                 <property name="horizontalSpacing">
                  <number>10</number>
                 </property>
                 <property name="verticalSpacing">
                  <number>14</number>
                 </property>
                 <property name="leftMargin">
                  <number>6</number>
                 </property>
                 <property name="topMargin">
                  <number>4</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="labelCoinControlAmountText">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Amount:</string>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="labelCoinControlAmount">
                   <property name="cursor">
                    <cursorShape>IBeamCursor</cursorShape>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::ActionsContextMenu</enum>
                   </property>
                   <property name="text">
                    <string notr="true">0.00 BTC</string>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="labelCoinControlLowOutputText">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Dust:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="labelCoinControlLowOutput">
                   <property name="cursor">
                    <cursorShape>IBeamCursor</cursorShape>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::ActionsContextMenu</enum>
                   </property>
                   <property name="text">
                    <string notr="true">no</string>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayoutCoinControl3">
                 <property name="horizontalSpacing">
                  <number>10</number>
                 </property>
                 <property name="verticalSpacing">
                  <number>14</number>
                 </property>
                 <property name="leftMargin">
                  <number>6</number>
                 </property>
                 <property name="topMargin">
                  <number>4</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="labelCoinControlFeeText">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Fee:</string>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="labelCoinControlFee">
                   <property name="cursor">
                    <cursorShape>IBeamCursor</cursorShape>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::ActionsContextMenu</enum>
                   </property>
                   <property name="text">
                    <string notr="true">0.00 BTC</string>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayoutCoinControl4">
                 <property name="horizontalSpacing">
                  <number>10</number>
                 </property>
                 <property name="verticalSpacing">
                  <number>14</number>
                 </property>
                 <property name="leftMargin">
                  <number>6</number>
                 </property>
                 <property name="topMargin">
                  <number>4</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="labelCoinControlAfterFeeText">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>After Fee:</string>
                   </property>
                   <property name="margin">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="labelCoinControlAfterFee">
                   <property name="cursor">
                    <cursorShape>IBeamCursor</cursorShape>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::ActionsContextMenu</enum>
                   </property>
                   <property name="text">
                    <string notr="true">0.00 BTC</string>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="labelCoinControlChangeText">
                   <property name="font">
                    <font>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Change:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="labelCoinControlChange">
                   <property name="cursor">
                    <cursorShape>IBeamCursor</cursorShape>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::ActionsContextMenu</enum>
                   </property>
                   <property name="text">
                    <string notr="true">0.00 BTC</string>
                   </property>
                   <property name="textInteractionFlags">
                    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacerCoinControl">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>800</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameSend">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font-weight:bold;</string>
            </property>
            <property name="text">
             <string>3. Send to BPQ address:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="textAddressTo">
            <property name="placeholderText">
             <string>Enter a BPQ address where to send bitcoins to</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="buttonSendToBpq">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Confirm the send action</string>
            </property>
            <property name="text">
             <string>S&amp;end</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/send</normaloff>:/icons/send</iconset>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
